integrations/os2sync/os2mo.py

```diff
@@ -198,6 +200,52 @@ def addresses_to_orgunit(orgunit, addresses):
             orgunit["Post"] = a["value"]
 
 
+def kle_to_orgunit(orgunit, kle):
+    """Collect kle uuids according to kle_aspect.
+
+    * Aspect "Udførende" goes into "Tasks"
+    * Aspect "Ansvarlig" goes into "ContactForTasks"
+
+    Example:
+
+        >>> orgunit={}
+        >>> kles = [
+        ...     {'uuid': 1, 'kle_aspect': [{'scope': 'UDFOERENDE'}]},
+        ...     {'uuid': 2, 'kle_aspect': [{'scope': 'ANSVARLIG'}]},
+        ...     {'uuid': 3, 'kle_aspect': [{'scope': 'ANSVARLIG'},
+        ...                                {'scope': 'UDFOERENDE'}
+        ...     ]}
+        ... ]
+        >>> kle_to_orgunit(orgunit, kles)
+        >>> orgunit
+        {'Tasks': [1, 3], 'ContactForTasks': [2, 3]}
+
+
+    Args:
+        orgunit: The organization unit to enrich with kle information.
+        kle: A list of KLEs.
+
+    Returns:
+        None
```

Comment:
En yderligere lille ting.

Jeg ville nok overveje at lave to funktioner.

En [pure function](https://en.wikipedia.org/wiki/Pure_function) som processere kles til de to lister, og en som har side-effekter ved at binde listerne til org_unit.

Det smarte ved det skulle være testbarhed af den rene funktion, og at side effekterne kan isoleres til beskidte funktioner.

Reply:
Det får du ikke - det kan du godt opgive!

Men nu er der test på!